contents/handbook/people/spending-money.md

```diff
@@ -35,6 +35,7 @@ You should default to using your company card in all cases - it has no transacti
 We now use Brex Empower. This allows the People and Ops team to effectively track overall spending habits of the company without waiting for end of month reports, with very little changes to how you use your Brex account.
 
 - Before spending money on your Brex card, you’ll need to log into your account either through the app or desktop, and assign a ‘spending budget’ to your card. The budget you pick from should relate to the transaction you are about to complete. For example, if you are buying a book for BookHog, this should come out of your ‘Books stipend’ budget. When you then complete your purchase, the money will be taken from that specific budget and you will easily be able to see how much of your budget you have remaining.
+- For subscriptions, you can create a virtual card and assign your subscriptions to that virtual card so all those charges will come from that spend limit and not your default card limit.
```

Comment:
```suggestion
- For subscriptions, you can [create a virtual card](https://dashboard.brex.com/p/wallet?expenses.wallet.view=all&walletCreateCard=true) and assign your subscriptions to that virtual card so all those charges will come from that spend limit and not your default card limit.
```

Reply:
Try to add links wherever possible - I _think_ this should work the same for anyone when logged in